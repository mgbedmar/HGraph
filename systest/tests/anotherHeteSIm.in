1
1
a1
0
1
a2
0
1
a3
0
1
a4
0
1
a5
0
1
p1
1
1
p2
1
1
p3
1
1
t1
2
1
t2
2
1
t3
2
1
c1
3
1
c2
3
4
a1
0
p1
1
4
a2
0
p1
1
4
a3
0
p2
1
4
a3
0
p3
1
4
a4
0
p3
1
4
a5
0
p3
1
4
t1
2
p1
1
4
t1
2
p3
1
4
t2
2
p2
1
4
t3
2
p3
1
4
c1
3
p1
1
4
c1
3
p2
1
4
c2
3
p3
1
0
0
0

