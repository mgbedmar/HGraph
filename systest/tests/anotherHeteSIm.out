Node (tipus: author, nom: a1) afegit.
Node (tipus: author, nom: a2) afegit.
Node (tipus: author, nom: a3) afegit.
Node (tipus: author, nom: a4) afegit.
Node (tipus: author, nom: a5) afegit.
Node (tipus: paper, nom: p1) afegit.
Node (tipus: paper, nom: p2) afegit.
Node (tipus: paper, nom: p3) afegit.
Node (tipus: term, nom: t1) afegit.
Node (tipus: term, nom: t2) afegit.
Node (tipus: term, nom: t3) afegit.
Node (tipus: conf, nom: c1) afegit.
Node (tipus: conf, nom: c2) afegit.
Aresta (tipusA: author, nomA: a1, tipusB: paper, nomB: p1) afegida.
Aresta (tipusA: author, nomA: a2, tipusB: paper, nomB: p1) afegida.
Aresta (tipusA: author, nomA: a3, tipusB: paper, nomB: p2) afegida.
Aresta (tipusA: author, nomA: a3, tipusB: paper, nomB: p3) afegida.
Aresta (tipusA: author, nomA: a4, tipusB: paper, nomB: p3) afegida.
Aresta (tipusA: author, nomA: a5, tipusB: paper, nomB: p3) afegida.
Aresta (tipusA: term, nomA: t1, tipusB: paper, nomB: p1) afegida.
Aresta (tipusA: term, nomA: t1, tipusB: paper, nomB: p3) afegida.
Aresta (tipusA: term, nomA: t2, tipusB: paper, nomB: p2) afegida.
Aresta (tipusA: term, nomA: t3, tipusB: paper, nomB: p3) afegida.
Aresta (tipusA: conf, nomA: c1, tipusB: paper, nomB: p1) afegida.
Aresta (tipusA: conf, nomA: c1, tipusB: paper, nomB: p2) afegida.
Aresta (tipusA: conf, nomA: c2, tipusB: paper, nomB: p3) afegida.
